services/net/reporting/ReportingManager.cs

```diff
@@ -350,7 +350,7 @@ private async Task<(API.Areas.Services.Models.ReportInstance.ReportInstanceModel
 
         // If we don't clear the prior report we must append new content to it.
         var currentSectionContent = new Dictionary<string, ReportSectionModel>();
-        if (reportInstanceModel != null && !report.Settings.Content.ClearOnStartNewReport)
+        if (reportInstanceModel != null && !reportInstanceModel.SentOn.HasValue && !report.Settings.Content.ClearOnStartNewReport)
```

Comment:
I'm not 100% certain but I think this should only currently include the prior content if it wasn't sent.